README.md

```diff
@@ -107,22 +107,22 @@ $ pgcopydb help
     prefetch   Stream JSON changes from the source database and transform them to SQL
     catchup    Apply prefetched changes from SQL files to the target database
     replay     Replay changes from the source to the target database, live
-  + sentinel   Maintain a sentinel table on the source database
+  + sentinel   Maintain a sentinel table
     receive    Stream changes from the source database
     transform  Transform changes from the source database into SQL commands
     apply      Apply changes from the source database into the target database
 
   pgcopydb stream sentinel
-    create  Create the sentinel table on the source database
-    drop    Drop the sentinel table on the source database
-    get     Get the sentinel table values on the source database
-  + set     Maintain a sentinel table on the source database
+    create  Create the sentinel table
+    drop    Drop the sentinel table
```

Comment:
We don't have these two commands anymore, I believe this PR should also remove the entries from this list.

Reply:
You are right, fixed. 👍